nf_core/module-template/modules/meta.yml

```diff
@@ -30,13 +30,13 @@ input:
 
 ## TODO nf-core: Add a description of all of the variables used as output
 output:
-  {% if has_meta -%}
+  #{% if has_meta -%} Only when we have meta
```

Comment:
was commenting the line here intended?

Reply:
Yes - Prettier breaks if it can't parse the YAML files. Sticking the Jinja templating stuff either in quotes or on comment lines means that the _template_ is still valid YAML and so prettier can format it.

Reply:
Note that this is a YAML comment and Jinja ignores it. If we wanted a Jinja comment that's done with `{# jinja here #}`. So the YAML comment `#` here has no effect.

Reply:
https://prettier.io/docs/en/ignore.html#yaml
I think this is what we need.